src/modules/token/fungible/hooks/queries/useGetHasUserAccount.js

```diff
@@ -0,0 +1,20 @@
+import { useInvokeQuery } from '@common/hooks';
+
+export const useGetHasUserAccount = ({ options = {}, config: customConfig = {}, } = {}) => {
+  const config = {
+    data: {
+      endpoint: 'token_hasUserAccount',
+      params: {
```

Comment:
To not deviate from our a bit strange query hook the pattern, like we do in some places, which have caused bugs (for example the search being open all the time). 

We should do:
```
    data: {
      endpoint: 'token_hasUserAccount',
      ...customConfig
    },
```